packages/yoroi-extension/app/components/wallet/assets/TokenList.js

```diff
@@ -272,7 +272,9 @@ function TokenItemRow({ avatar, name, id, amount, isTotalAmount }: TokenItemRowP
           <CopyToClipboardText text={id}>{truncateAddressShort(id)}</CopyToClipboardText>
         </Typography>
       }
-      thirdColumn={<Typography fontWeight="500">{amount}</Typography>}
+      thirdColumn={
+        <Typography fontWeight="500">{new Intl.NumberFormat().format(amount)}</Typography>
```

Comment:
@neuodev , the number format must be dependent on the selected language. The formatting of the numbers must be matching the selected locale, for now. Later we might think on having it as a separate option, but not right now.

Make sure you resolve the currently selected language and pass it into the `NumberFormat` constructor.